src/main/javac/org/apache/brooklyn/tosca/a4c/platform/transformer/converters/ToscaComputeLocToVanillaConverter.java

```diff
@@ -0,0 +1,66 @@
+package org.apache.brooklyn.tosca.a4c.platform.transformer.converters;
+
+import alien4cloud.model.topology.NodeTemplate;
+import com.google.common.collect.Iterables;
+import org.apache.brooklyn.api.entity.EntitySpec;
+import org.apache.brooklyn.api.location.Location;
+import org.apache.brooklyn.api.location.LocationDefinition;
+import org.apache.brooklyn.api.mgmt.ManagementContext;
+import org.apache.brooklyn.core.entity.trait.Startable;
+import org.apache.brooklyn.entity.software.base.VanillaSoftwareProcess;
+import org.apache.brooklyn.util.collections.MutableMap;
+import org.apache.brooklyn.util.exceptions.Exceptions;
+import org.apache.brooklyn.util.guava.Maybe;
+import org.apache.brooklyn.util.text.Strings;
+import org.apache.brooklyn.util.time.Duration;
+
+import java.util.Map;
+
+public class ToscaComputeLocToVanillaConverter extends ToscaComputeToVanillaConverter {
+
```

Comment:
This converter allows to create a EntitSpec<TomcatServer> from a `tosca.nodes.Tomcat` template.
However, the custom type `tosca.nodes.Tomcat` is not the best option. As I understood previously, we should use custom-types for representing the Brooklyn entities.